### ISSUE TITLE
error: no such remote ref 093e4dee3cfa54b176142ca9840697154cbd97df

### ISSUE THREAD

**Issue by lfaoro** [IBM-Swift/Kitura#570]
![image](https://cloud.githubusercontent.com/assets/3834345/16497990/eb433ed6-3ef9-11e6-9f6c-359bfefc0831.png)

**Comment by lfaoro** [comment]
```Swift
import PackageDescription

let package = Package(
    name: "asm-layer",
    dependencies: [
         .Package(url: "https://github.com/IBM-Swift/Kitura.git", majorVersion: 0, minor: 19)
    ])
```

**Comment by ianpartridge** [comment]
Can you provide a GitHub project I can clone to recreate this problem?

**Comment by vadimeisenbergibm** [comment]
The issue is related to git 2.9.0. We will resolve it soon. Alternatively, please try another git version.

**Comment by vadimeisenbergibm** [comment]
It seems our plan to fix the problem with git 2.9.0 will not work. Please try another git version, e.g. 2.9.1.

**Comment by ianpartridge** [comment]
Closing as 2.9.1 fixes this.
